ods_ci/tests/Resources/OCP.resource

```diff
@@ -65,11 +65,8 @@ Is RHODS Self-Managed
     ...                 in the cluster (e.g., usually on OSD, ROSA).
     ...                 Returns ${TRUE} if RHODS Self-Managed is installed or if PRODUCT=ODH
     IF  "${PRODUCT}" == "ODH"  RETURN     ${TRUE}
-    ${is_managed} =    Run Keyword And Return Status    OpenshiftLibrary.Oc Get
-    ...                                                      kind=CatalogSource
-    ...                                                      name=addon-managed-odh-catalog
-    ...                                                      namespace=${OPERATOR_NAMESPACE}
-    Run Keyword And Return    Evaluate    not ${is_managed}
+    ${rc}=    Run And Return Rc    oc get catalogsource addon-managed-odh-catalog -n ${OPERATOR_NAMESPACE}
```

Comment:
Would be nice to put a comment above this line so we assure that nobody else will change it back to use OpenshiftLibrary again in the future :slightly_smiling_face:

Reply:
I've added it